src/appComponents/tracks/TracksController.js

```diff
@@ -1,56 +1,65 @@
 /**
  * Created by championswimmer on 29/5/15.
  */
+/* exported error */
 
-var tracksModule = angular.module('oe.tracks', ['ui.router']);
+const tracksModule = angular.module("oe.tracks", ["ui.router"]);
+
+tracksModule.config(["$stateProvider", function($stateProvider) {
+
+  $stateProvider.state("tracks", {
+    "url": "/tracks",
+    "templateUrl": "appComponents/tracks/tracks.html",
+    "controller": "TracksController"
+  });
 
-tracksModule.config(['$stateProvider', function($stateProvider) {
-    $stateProvider.state('tracks', {
-        url: '/tracks',
-        templateUrl: 'appComponents/tracks/tracks.html',
-        controller: 'TracksController'
-    });
 }]);
 
-tracksModule.controller('TracksController', 
-	['$mdDialog','$sessionStorage', '$rootScope', 'ApiJsonFactory',
-        function($mdDialog, $sessionStorage, $rootScope, ApiJsonFactory) {
-		var tc = this;
-        if ( $sessionStorage.tracks === null ||
-            typeof($sessionStorage.tracks) == 'undefined')
-        {
+tracksModule.controller("TracksController", ["$mdDialog","$sessionStorage", "$rootScope", "ApiJsonFactory",
+        function ($mdDialog, $sessionStorage, $rootScope, ApiJsonFactory) {
+          var tsc = this;
+
+          if ($sessionStorage.tracks === null || 
+            typeof $sessionStorage.tracks === "undefined")
+         {
+
             $sessionStorage.tracks = [];
-        }
-        tc.Tracks = $sessionStorage.tracks;
-        tc.Sessions = $sessionStorage.sessions;
+
+          }
+        tsc.Tracks = $sessionStorage.tracks;
+        tsc.Sessions = $sessionStorage.sessions;
         
 
-        if (tc.Tracks.length === 0) {
-            ApiJsonFactory.getJson('tracks')
-                .then(function (response) {
-                    tc.Tracks = response.data.tracks;
-                    $sessionStorage.tracks = tc.Tracks;
-                }, function (error) {
-                    console.error(error);
-                });
+        if (tsc.Tracks.length === 0) {
+
+          ApiJsonFactory.getJson("tracks")
+            .then(function (response) {
+                tsc.Tracks = response.data.tracks;
+                $sessionStorage.tracks = tsc.Tracks;
+            }, function error (error) {
+                    //console.error(error);
+            });
 
-            ApiJsonFactory.getJson('sessions')
-                .then(function (response) {
-                    tc.Sessions = response.data.sessions;
-                    $sessionStorage.sessions = tc.Sessions;
-                }, function (error) {
-                    console.error(error);
+          ApiJsonFactory.getJson("sessions")
+            .then(function (response) {
+
+                tsc.Sessions = response.data.sessions;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 18 space characters but found 20.](https://www.codacy.com/app/triplez/open-event-webapp/file/2418144097/issues/source?bid=3233857&fileBranchId=3237740#l46)